ios/AppLovinMAX.m

```diff
@@ -44,9 +44,10 @@ @interface AppLovinMAX()
 @property (nonatomic, strong, nullable) NSNumber *locationCollectionEnabledToSet;
 @property (nonatomic, strong) NSMutableDictionary<NSString *, NSString *> *extraParametersToSet;
 
-@property (nonatomic, strong, nullable) NSNumber *consentFlowEnabledToSet;
+@property (nonatomic, strong, nullable) NSNumber *termsAndPrivacyPolicyFlowEnabledToSet;
```

Comment:
We need to support both old `consentFlow` and `termsAndPrivacyPolicyFlow`. Right now, we automatically move pubs to the new flow, if they are on the old flow.

Reply:
Added `AppTrackingStatus`, also kept the old consent flow on iOS.   Android does not support the old consent flow.    Please review.